src/skia/SamplingOptions.cpp

```diff
@@ -0,0 +1,16 @@
+#include "common.h"
+#include <include/core/SkSamplingOptions.h>
+
+void initSamplingOptions(py::module& m) {
+    py::class_<SkSamplingOptions> samplingoptions(m, "SamplingOptions");
+
+        py::enum_<SkFilterMode>(m, "FilterMode")
+        .value("kNearest", SkFilterMode::kNearest,
+            "single sample point (nearest neighbor)")
+        .value("kLinear", SkFilterMode::kLinear,
+            "interporate between 2x2 sample points (bilinear interpolation)")
+        .export_values();
+
+    samplingoptions
```

Comment:
Is there a test case for this?

Reply:
No, but the tests have been updated with new 'skia.SamplingOptions()' arguments where they are needed. The enum values above is needed as a new argument to one of the surface/canvas(?) Routines. I don't want to add new code to access the internals or the non-default constructors of this class. (If nobody asks for it, a bare default no-args constructor would probably make most people happy)

Reply:
I can add a quick "is_instance" test if you insist?

Reply:
I have staged a basic isinstance test over at https://github.com/HinTak/skia-m1xx-python/pull/1 . Will cherry-pick when CI passes over there.